microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,623 @@
+   1. Change **Width** & **Height** to 75, 30 respectively:
+
+   ![Updated Back Button properties in Unity's Inspector](./hololens2-images/unity_backbutton_inspector.png)
+
+1. Update the second button's properties by selecting it in the **Hierarchy** pane, then making the following changes in the **Inspector** pane:
+   1. Change **Name** to "Go Button"
+   1. Change **PosX**, **PosY**, **PosZ** to 242, 178, -5, respectively.
+   1. Change **Width** & **Height** to 75, 30 respectively:
+
+   ![Updated Go Button Properties in Unity's Inspector](./hololens2-images/unity_gobutton_inspector.png)
+
+1. Update the WebView's properties by selecting it in the **Hierarchy** pane, then making the following changes in the **Inspector** pane:
+   1. Change **PosX**, **PosY**, **PosZ** to 0, -16, -5, respectively.
+   1. Change **Scale** for X, Y, Z to 570, 340, 1 respectively:
+
+   ![Updated WebView Properties in Unity's Inspector](./hololens2-images/unity_webview_transform_inspector.png)
+
+1. Add a background image to the **Canvas**:
+   1. Select the Canvas in the **Hierarchy** pane.
+   1. Click the **Add Component** button at the bottom of the **Inspector** pane.
+   1. Type **Image** and then select the top result in the list.
+   1. Click the **Color** well for the image and then choose a background color. In our example, we chose a gray color. (This is simply to show some contrast between the different controls in the canvas.)
+
+   ![Change Back Button text in Unity's Inspector](./hololens2-images/unity_canvasimage_inspector.png)
+
+1. In the **Hierarchy** pane, under **Back Button**, select **Text (TMP)**.  Then in the **Inspector** pane, change the **Text Input** to **Back**:
+
+   ![Change Go Button text in Unity's Inspector](./hololens2-images/unity_backbutton_text.png)
+
+1. Repeat the above process for **Go Button**, substituting **Go** as the text.
+
+   You should now have a scene which looks similar to the following:
+
+   ![The current scene in Unity's Scene view](./hololens2-images/unity_completed_scene.png)
+
+1. Connect WebViewBrowser prefabs:
+   1. Select the WebViewBrowser.
+   1. Drag **Back Button** from the Hierarchy pane into the **Back Button** variable field for the WebView Browser in the Inspector.
+   1. Drag **Go Button** from the Hierarchy pane into the **Go Button** variable field for the WebView Browser in the Inspector:
+
+   ![The WebView Browser Prefab with assigned variables in Unity's Inspector](./hololens2-images/unity_webview_connect_prefabs.png)
+
+1. Test the scene in the Editor. Now that everything is wired up in your scene, you should be able to play the scene inside Unity. Try testing out the scene by entering a new URL into the field<!--todo: what field?  give name of field/UI control label, or some description/location in window; the Address bar?--> (must be a complete URL), then pressing the **Go** button. Make sure the **Back** button works, as well.
```

Comment:
@michaelfarnsworth idk context.  What field? 
Give name of field/UI control label, or some description/location in window; the Address bar?

Reply:
need screenshot here?

Reply:
It's the AddressField, which is the text input control shown just above in ./hololens2-images/unity_completed_scene.png.

Reply:
gh rendered: scroll up from
https://github.com/michaelfarnsworth/edge-developer/blob/webview2-hololens-getting-started/microsoft-edge/webview2/get-started/hololens2.md#step-7---build-and-deploy-the-hololens-2-app-with-webview2

Reply:
Right, that's the placeholder text. We can refer to as that, if you want.

Reply:
locally changed to
```
Try testing out the scene by entering a new URL into the **Enter text*** field (must be a complete URL), and then clicking the **Go** button.
```
you can close this thread